lms/djangoapps/program_enrollments/rest_api/v1/tests/test_views.py

```diff
@@ -231,7 +231,10 @@ def create_program_enrollments(self):
         for i in range(2, 4):
             user_key = 'user-{}'.format(i)
             ProgramEnrollmentFactory.create(
-                program_uuid=self.program_uuid, curriculum_uuid=self.curriculum_uuid, external_user_key=user_key,
+                program_uuid=self.program_uuid,
+                curriculum_uuid=self.curriculum_uuid,
+                user= UserFactory.create(username='student-{}'.format(i), email='email-{}'.format(i)),
```

Comment:
`E251 unexpected spaces around keyword / parameter equals`. The space before `UserFactory` should be removed